### PR TITLE
Fixes #24327 - Update outdated references in README

### PR DESCRIPTION
**Pull request by johnpmitsch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Katello
 
-[![Build Status](https://ci.theforeman.org/buildStatus/icon?job=test_katello)](https://ci.theforeman.org/job/test_katello)
+[![Build Status](https://ci.theforeman.org/buildStatus/icon?job=katello-nightly-release)](https://ci.theforeman.org/job/katello-nightly-release/)
```
Comment (johnpmitsch) [inline README.md]: Is this the right jenkins job?

Comment (jlsherrill) [reply README.md]: This would technically be the release job, but i think that is okay!

File: README.md [hunk 2]
```diff
@@ -19,18 +19,18 @@ Linux based systems.
...
 
 The most common way to set up Katello for development is to use
-[katello-deploy](https://github.com/Katello/katello-deploy#development-deployment).
-This will set up a Vagrant instance with the Katello codebase checked out. You
-can also run `setup.rb` directly with katello-deploy if you prefer to not use
-Vagrant.
+[forklift](https://github.com/theforeman/forklift).
+This will set up a virtual machine with the Katello codebase checked out.
+Please use the forklift documentation found in the repository for how
+to get started with forklift.
```
Comment (beav) [inline README.md]: :+1:

**Comment by beav** [comment]
krul

**Pull request merged**